scripts/retry-tracker.md

```diff
@@ -0,0 +1,44 @@
+# Retry-Tracker script
+<!-- TOC -->
+* [Retry-Tracker script](#retry-tracker-script)
+  * [What it does?](#what-it-does)
+  * [When we use it?](#when-we-use-it)
+  * [How it Works?](#how-it-works)
+  * [Parameters](#parameters)
+  * [Conclusion/QuickGuide:](#conclusionquickguide)
+<!-- TOC -->
+## What it does?
+
+The Retry-Tracker script is used to retry failed commits to the OpenURLTracker in DSpace. The OpenURLTracker is used to
+track requests made to external systems, such as DOI lookup services, and store metadata about those requests in the
```

Comment:
We should also mention this script is useful in [exchanging usage statistics information with IRUS](https://wiki.lyrasis.org/display/DSDOC7x/Exchange+usage+statistics+with+IRUS).  That's what the OpenUrlTracker was created for (see https://github.com/DSpace/DSpace/pull/2664)